pkg/comp-functions/functions/vshnnextcloud/files/vshn-nextcloud.config.php

```diff
@@ -0,0 +1,4 @@
+<?php
+$CONFIG = array (
+  'check_data_directory_permissions' => false,
```

Comment:
I guess this file will be merged with the default config.php right?

Reply:
Yes